Update README: redirect users to the discussion tab (#526)
online-ml/river: README.md

## 👍 Contributing

Feel free to contribute in any way you like, we're always open to new ideas and approaches. 

> We have three main ways to users get involved with `river`:

Reviewer: ```suggestion
There are three ways for users to get involved:
```

## 👍 Contributing

> Feel free to contribute in any way you like, we're always open to new ideas and approaches. You can also take a look at the [issue tracker](https://github.com/online-ml/river/issues) and the [discussions](https://github.com/online-ml/river/discussions) to see if anything takes your fancy. Please check out the [contribution guidelines](https://github.com/online-ml/river/blob/master/CONTRIBUTING.md) if you want to bring modifications to the code base. You can view the list of people who have contributed [here](https://github.com/online-ml/river/graphs/contributors).

Reviewer: I think you should put a link to the board and not the discussions, no? The idea for the board is that contributers can take a look at what needs doing and hasn't been assigned yet. The discussions are there to discuss new ideas. What do you think?

Author: Sounds good to me. I wanted to point to the discussions somewhere since we already have that badge for the roadmap.

But that works too. Let me change it.

Reviewer: You're right it's already linked somewhere, but we may as well overcommunicate a little so people don't miss it :)
